dataland-frontend/src/components/general/BulkDataResponseDialog.vue

```diff
@@ -0,0 +1,125 @@
+<template>
+  <div data-test="bulkDataResponseDialog">
```

Comment:
Check this dialog once communication with the backend works

Reply:
This is part of the dialog that is turned off. Will be moved to different branch.

Reply:
ok, can you ping me once this is done?

Reply:
Moved to branch `DALA-3177_extend_data_requests_PHASE_2`
Description of work here: https://github.com/d-fine/DatalandInternal/wiki/DALA%E2%80%903177-%E2%80%90-Phase-2-%E2%80%90-Identifiers-Request-Dialog

Reply:
ok thank you